### PR TITLE
New release v4.1.3

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,33 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+- Switched from `Coveralls` to `Codecov` (monorepo preparation, coverage
+  reports on PRs),
+  PR [#646](https://github.com/ethereumjs/ethereumjs-vm/pull/646)
+- Added nightly `StateTests` runs,
+  PR [#639](https://github.com/ethereumjs/ethereumjs-vm/pull/639)
+- Run consensus tests on `MuirGlacier`,
+  PR [#648](https://github.com/ethereumjs/ethereumjs-vm/pull/648)
+
+[4.1.3]: https://github.com/ethereumjs/ethereumjs-vm/compare/v4.1.2...v4.1.3
```
Comment (alcuadrado) [inline CHANGELOG.md]: I'm not sure if this was already discussed, but just in case: These links have to be updated when migrating to a monorepo.

Comment (evertonfraga) [reply CHANGELOG.md]: I already took care of that link migration in particular and some others as well :)

**Comment by holgerd77** [comment]
This needs to be re-approved once tests are through.

File: CHANGELOG.md (hunk 1 above)
Comment (evertonfraga) [reply CHANGELOG.md]: the issue I found with tag diff link is that we can’t filter it by subdirectory. But that’s not too bad IMO.

**Comment by holgerd77** [comment]
Just released as [v4.1.3](https://github.com/ethereumjs/ethereumjs-vm/releases/tag/v4.1.3) on npm.

**Comment by holgerd77** [comment]
@davidmurdoch (Truffle/Ganache) @yann300 (Remix) @javier-tarazaga (Ethereum Studio)

Hi guys, we had a critical bug in our `MuirGlacier` release `v4.1.2` which prevented the VM to work properly when instantiated with the `MuirGlacier` HF. An update is necessary to have a working VM installation. Please let me know if you have got questions!

Side note: anyone else reading this and wants to be added to (or removed from) this informal VM users push-notification list feel free to drop a note below!

**Comment by yann300** [comment]
cc @Aniket

File: CHANGELOG.md (hunk 1 above)
Comment (alcuadrado) [reply CHANGELOG.md]: That's great :)

About the subdirectories, that's not terrible. Also, I have a feeling that Github will have better support for monorepos in the near future. Monorepos used to go against their business model, and now not so much.

**Pull request merged**